cfme/tests/test_rest_no_provider.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.utils.wait import wait_for
+
+pytestmark = [test_requirements.rest, pytest.mark.tier(1)]
+
+
+@pytest.fixture
+def change_company_name(appliance):
+    old_company_name = appliance.advanced_settings["server"]["company"]
+    updated_company_name = "REST Company Name"
+
```

Comment:
These blank lines can be removed

Reply:
Hey @sshveta, there are a few test cases that do not require a provider. Since the file `test_rest.py` is marked with a global provider marker, the provider will be set up for all the tests under it. I could use `has_no_providers` fixture, but then again it will first set up the provider and then remove it, which would take even more time. The majority of the tests under the file require a provider, so I do not want to remove the global provider either, which is why I created a separate file for all the tests that do not require providers.

I agree that the file name could be better, so if you have a suggestion, let me know :)

Reply:
How about `test_providerless_rest.py`? Then have a small comment at the top `test_rest.py` and this new file.